application/common/models/Password.php

```diff
@@ -114,7 +114,7 @@ private function expires(): Closure
         return function() {
             $lifespan = Yii::$app->params['passwordLifespan'];
 
-            return MySqlDateTime::format(strtotime($lifespan, strtotime($this->created_utc)));
+            return MySqlDateTime::formatDate(strtotime($lifespan, strtotime($this->created_utc)));
```

Comment:
If PHP's default timezone is set to something other than UTC, this may not give you back the point-in-time that you expect. Giving `strtotime(...)` a string that doesn't specify a timezone results in it assuming the given time is in PHP's default timezone. I think we have that set to UTC for our PHP apps, so this may be fine.